doc/user/content/transform-data/patterns/time-travel-queries.md

```diff
@@ -0,0 +1,153 @@
+
+* Newly produced historical data is retained for the new, shorter history
+  retention period.
+
+* Historical data outside the new, shorter history retention period is no longer
+  retained. If you subsequently increase the history retention period again,
+  the older historical data may already be unavailable.
+
+### Resource utilization
+
+It's important to note that increasing the history retention period for an
+object will lead to increased resource utilization in Materialize.
+
+* **For sources, tables and materialized views:** increasing the history
+    retention period for these objects increases the amount of historical data
+    that is retained in the storage layer. You can expect storage resource
+    utilization to increase, which may incur additional costs.
+
+* **For indexes:** increasing the history retention period for an index
+    increases the amount of historical data that is retained in memory in the
+    cluster maintaining the index. You can expect memory resource utilization
+    for that cluster to go up, which might require you to size up your cluster
+    and consume additional compute credits.
+
+### Clean-up
+
+The history retention period represents the minimum amount of historical data
+guaranteed to be retained by Materialize. History clean-up is processed in the
+background, so older history may be accessible for the period of time between
+when it falls outside the retention period and when it is cleaned up.
+
+### Best practices
+
+Given the additional memory costs associated with increasing the history
+retention period on indexes, if you don't need the performance benefits of an
+index - in particular for the use case of [durable subscriptions](/sql/subscribe#durable-subscriptions) -
+you should consider creating a materialized view for your subscription query
+and configure the history retention period on that materialized view
+instead.
+
+Similarly, because of the increased storage costs and processing time for the
+additional historical data on whichever object has an increased history
+retention period, you should consider configuring history retention period on
+the index or materialized view directly powering the subscription, rather than
+all the way through the dependency chain from the source to the index or
+materialized view.
+
+## Examples
+
+### Enabling history retention
+
+To set a history retention period for an object, use the `RETAIN HISTORY`
+option, which accepts positive [interval](/sql/types/interval/) values
+(e.g. `'1hr'`):
+
+```sql
+CREATE MATERIALIZED VIEW winning_bids
+WITH (RETAIN HISTORY FOR '1hr') AS
+SELECT auction_id,
+       bid_id,
+       item,
+       amount
+FROM highest_bid_per_auction
+WHERE end_time < mz_now();
+```
+
+To adjust the history retention period for an object, use `ALTER`:
+
+```sql
+ALTER MATERIALIZED VIEW winning_bids SET (RETAIN HISTORY FOR '2hr');
+```
+
+<!-- TODO(maddyblue): replace this paragraph with a mention of the catalog
```

Comment:
This exists now https://materialize.com/docs/sql/system-catalog/mz_internal/#mz_history_retention_strategies

Reply:
The PR just barely missed the release cutoff b/c it was cut early for the onsite :( I'll put a warn if unreleased.

Reply:
aww, rip